br/pkg/lightning/backend/local/localhelper.go

```diff
@@ -476,12 +479,6 @@ func (local *local) checkScatterRegionFinishedOrReScatter(ctx context.Context, r
 		if respErr.GetType() == pdpb.ErrorType_REGION_NOT_FOUND {
 			return true, nil
 		}
-		// don't return error if region replicate not complete
-		// TODO: should add a new error type to avoid this check by string matching
-		matches, _ := regexp.MatchString("region \\d+ is not fully replicated", respErr.Message)
-		if matches {
-			return false, nil
-		}
 		return false, errors.Errorf("get operator error: %s", respErr.GetType())
```

Comment:
should include `respErr.Message` in the returned error

Reply:
Fixed in 42bf379b79afc02648a84b44664e295c56aef58a.